services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -260,6 +263,18 @@ async def _start_dynamic_service(
             service_version=service_version,
         )
 
+        # Get wallet information
+        wallet_info = None
+        project_wallet = await get_project_wallet(request.app, project_id=project_uuid)
+        if project_wallet:
```

Comment:
```suggestion
        if project_wallet:=await get_project_wallet(request.app, project_id=project_uuid):
```

Reply:
and minor question not for now, what about passing the user_id to get_project_wallet and checking the permission right away?